src/test/java/HomeWork17.java

```diff
@@ -0,0 +1,65 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+
+
+    public class HomeWork17 extends BaseTest {
+
+        @Test
+        public void addSongToPlaylist() throws InterruptedException {
+
+            navigateToPage();
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/227728309-1504e736-5da8-4741-bad2-dc1f75412e97.png)